### ISSUE TITLE
Add option to deactivate "dead" blog

### ISSUE THREAD

**Issue by tdziurko** [tdziurko/jvm-bloggers#66]

**Comment by matek2305** [comment]
Should we make some kind of panel with blog list where we would have last post date and deactivate/active option? Something like current moderation panel but for blogs instead of posts.

**Comment by tdziurko** [comment]
Yes, page similar to "Moderation" but with blogs allowing to deactivate them. Deactivated blog should be no longer checked for new blog posts and blog_posts from such blogs should not be included in newsletter.

**Comment by matek2305** [comment]
Ok, so I will try to do this then ;)

**Comment by tdziurko** [comment]
Maybe it is a good idea to split it into two tasks:
1. Ability to deactivate blog (this one)
2. Ignoring deactivated blogs and blog posts in newsletter generation and rss fetching process
?

**Comment by matek2305** [comment]
Ok, so I will add issue for 2nd task.

**Comment by goostleek** [comment]
I'm wondering how administrator can determine that given blog is dead. It's not possible without investigating application logs. Maybe we should introduce some logic to mark the blog as dead automatically if the blog cannot be accessed by fetcher after several (configurable at application level) subsequent fetcher executions?

**Comment by adamdec** [comment]
404 = dead ;)

**Comment by tdziurko** [comment]
Maybe we should store status of last X attempts to fetch rss from given blog as OK, ERROR and display it next to the blog in data table?
Manual disabling might be helpful when user wants to disable completely working blog.

**Comment by goostleek** [comment]
I would rather opt to implement automatic logic to disable problematic blog with option allowing manual disabling from admin panel. I always prefer doing something automatically if sth. is simple, adding human factor is always a risk that somebody will do sth. wrong by a mistake.

**Comment by matek2305** [comment]
I opened pr #163 with manual disabling option. Automatic logic should be new task IMO, also can you @tdziurko explain a little bit more about storing fetch status, should it look like something like this?
![image](https://cloud.githubusercontent.com/assets/7036513/13906580/ec680942-eed9-11e5-9743-49fe3bd51e6e.png)
